proc/proc.go

```diff
@@ -189,9 +189,15 @@ func parseKernelModuleLine(line string) (kernelModule, error) {
 }
 
 func parseAddress(addressStr string) (uint64, error) {
-	address, err := strconv.ParseUint(strings.TrimPrefix(addressStr, "0x"), 16, 64)
+	addrParts := strings.SplitAfter(addressStr, " ")
```

Comment:
We can simplify by changing L169 into
```
parts := strings.SplitN(line, " ", 7)
```
Then you won't need to call SplitAfter and TrimSpace here.

Reply:
good point - this simplifies things.